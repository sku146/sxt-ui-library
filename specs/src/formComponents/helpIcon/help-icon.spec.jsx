import React from 'react';
import renderer from 'react-test-renderer';
import HelpIcon from 'src/formComponents/helpIcon/help-icon';

describe("Help Icon", () => {

  let defaultProps = {
    name: 'test label name',
  };

  const RendererComponent = (props) => {
    return renderer.create(<HelpIcon {...props} />);
  };

  it('component should render as expected', ()=> {
    const component = RendererComponent(defaultProps);
    expect(component.toJSON()).toMatchSnapshot();
  });
});
