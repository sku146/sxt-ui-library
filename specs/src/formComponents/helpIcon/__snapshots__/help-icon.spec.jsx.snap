// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Help Icon component should render as expected 1`] = `
<i
  aria-hidden="true"
  className="help circle outline icon"
  data-bdd-selector="test label nameHelpIcon"
/>
`;
